modules-js/react-fleet/src/layouts/AppLayout.tsx

```diff
@@ -45,7 +45,11 @@ export default function AppLayout({
 
           {secondaryNav}
 
-          {breadcrumbNav}
+          {secondaryNav ? (
+            <>{breadcrumbNav}</>
+          ) : (
+            <div className="p-t300">{breadcrumbNav}</div>
```

Comment:
I think this is unnecessary now?

Reply:
My understanding was that the Seal sometimes will overlap with the breadcrumb text when the secondary nav isn’t present, which is a separate concern from the scrolling behavior.